addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -72,6 +73,54 @@ export class AnalyticDistribution extends Component {
         this.allPlans = [];
         this.lastAccount = this.props.account_field && this.props.record.data[this.props.account_field] || false;
         this.lastProduct = this.props.product_field && this.props.record.data[this.props.product_field] || false;
+
+        this.activeTag = null;
+        this.selectCreateIsOpen = false;
+        const selectCreate = useSelectCreate({
+            resModel: 'account.analytic.account',
+            activeActions: useActiveActions({
+                fieldType: "many2many",
+                crudOptions: {
+                    link: true,
+                    create: false,
```

Comment:
Maybe we can allow this to be true? Probably in another task though...
Not sure if it is useful or if most of the tags are already created automatically through the Project app etc.